src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CaseRepository.java

```diff
@@ -84,7 +84,7 @@ Claim updateClaimSubmissionOperationStatus(
         ClaimSubmissionOperationIndicators indicators,
         CaseEvent caseEvent);
 
-    void updateClaimState(String authorisation, Long claimId, ClaimState state);
+    void updateClaimStateToOpen(String authorisation, Long claimId, ClaimState state);
```

Comment:
If `state` is guaranteed to be `OPEN` then we don't need the parameter; otherwise the method name change is inappropriate.

Reply:
state is not guaranteed to be open. we check that in the implementation. The name change was to make it clear that we are changing the state to Open.

Reply:
As discussed offline: now I understand the purpose of this method, I'd like to see the `state` parameter renamed to `currentState` or `previousState` to make it clear that it's not specifying the target state.